config/runtimes/caikit-tgis-template.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  labels:
+    opendatahub.io/dashboard: 'true'
+    opendatahub.io/ootb: 'true'
+  annotations:
+    description: Caikit is an AI toolkit that enables users to manage models through a set of developer friendly APIs. It provides a consistent format for creating and using AI models against a wide variety of data domains and tasks.
+    openshift.io/provider-display-name: Red Hat, Inc.
+    tags: rhods,rhoai,kserve,servingruntime
+    template.openshift.io/documentation-url: https://github.com/opendatahub-io/caikit-tgis-serving
+    template.openshift.io/long-description: This template defines resources needed to deploy caikit-tgis-serving servingruntime with Red Hat Data Science KServe for LLM model
+    template.openshift.io/support-url: https://access.redhat.com
+    opendatahub.io/modelServingSupport: '["single"]'
+    opendatahub.io/apiProtocol: 'REST'
+  name: caikit-tgis-serving-template
+objects:
+  - apiVersion: serving.kserve.io/v1alpha1
+    kind: ServingRuntime
+    metadata:
+      name: caikit-tgis-runtime
+      annotations:
+        openshift.io/display-name: Caikit TGIS ServingRuntime for KServe
+        opendatahub.io/recommended-accelerators: '["nvidia.com/gpu"]'
+      labels:
+        opendatahub.io/dashboard: 'true'
+    spec:
+      annotations:
+        prometheus.kserve.io/port: '3000'
```

Comment:
I guess the correct annotations are just `prometheus.io`.

Reply:
done.

Reply:
This is pretty interesting, what difference does it make with `prometheus.kserve.io`?

Reply:
`prometheus.kserve.io` is designed for the kserve + knative metrics aggregation. That's not officially supported by Red Hat Serverless. So we are using metrics aggregation with kserve + istio which requires the `prometheus.io` annotations. This is where our Kserve installation differs from upstream